test/models/repository_test.rb

```diff
@@ -312,6 +312,19 @@ def test_search_redhat
       refute_includes repos, @fedora_17_x86_64
       refute_includes repos, @puppet_forge
     end
+
+    def test_search_content_label
+      @content_id = 'somecontent-123'
```

Comment:
we can use local variables instead of instance variables here :)

Reply:
Updated. :+1: